Spigot-Server-Patches/0696-Horse-Api.patch

```diff
@@ -0,0 +1,110 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Owen1212055 <23108066+Owen1212055@users.noreply.github.com>
+Date: Sat, 8 May 2021 19:01:48 -0400
+Subject: [PATCH] Horse Api
+
+
+diff --git a/src/main/java/net/minecraft/world/entity/animal/horse/EntityHorseAbstract.java b/src/main/java/net/minecraft/world/entity/animal/horse/EntityHorseAbstract.java
+index 1e41c45af6dbcf097d7d6104e63db637f199301a..10174015e513feef236b54a2be98b41be7919531 100644
+--- a/src/main/java/net/minecraft/world/entity/animal/horse/EntityHorseAbstract.java
++++ b/src/main/java/net/minecraft/world/entity/animal/horse/EntityHorseAbstract.java
+@@ -131,7 +131,7 @@ public abstract class EntityHorseAbstract extends EntityAnimal implements IInven
+     protected boolean t(int i) {
+         return ((Byte) this.datawatcher.get(EntityHorseAbstract.bz) & i) != 0;
+     }
+-
++    public boolean getHorseFlag(int mask) {return t(mask);} // Paper - OBFHELPER
+     protected void d(int i, boolean flag) {
+         byte b0 = (Byte) this.datawatcher.get(EntityHorseAbstract.bz);
+ 
+@@ -142,6 +142,7 @@ public abstract class EntityHorseAbstract extends EntityAnimal implements IInven
+         }
+ 
+     }
++    public void setHorseFlag(int mask, boolean value) {d(mask, value);} // Paper - OBFHELPER
+ 
+     public boolean isTamed() {
+         return this.t(2);
+@@ -179,10 +180,12 @@ public abstract class EntityHorseAbstract extends EntityAnimal implements IInven
+     public boolean eZ() {
+         return this.t(16);
+     }
++    public boolean isEatingGrass() {return eZ();} // Paper - OBFHELPER
+ 
+     public boolean fa() {
+         return this.t(32);
+     }
++    public boolean isRearing() {return fa();} // Paper - OBFHELPER
+ 
+     public boolean hasReproduced() {
+         return this.t(8);
+@@ -653,10 +656,13 @@ public abstract class EntityHorseAbstract extends EntityAnimal implements IInven
+         }
+ 
+     }
++    public void setEating(boolean open) {setHorseFlag(64, open);} // Paper - OBFHELPER
++    public boolean isEating() {return getHorseFlag(64);} // Paper - OBFHELPER
```

Comment:
Not sure I'd call these obfhelpers, but I also didn't see how the team handles such additions normally.